util/test/management.go

```diff
@@ -0,0 +1,91 @@
+package test
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+// Server management API server for testing
+type Server struct {
+	HTTPServer *httptest.Server
+
+	t       *testing.T
+	rootDir string
+	expCode int
+}
+
+// NewTestManagementServer instantiates a new management API server backend for testing
+func NewTestManagementServer(t *testing.T) *Server {
+	fs, fsCleanup, err := MakeTempFS("", "opa_management", map[string]string{})
+	if err != nil {
+		t.Fatalf("unexpected error %v", err)
+	}
+
+	server := &Server{
+		t:       t,
+		rootDir: fs,
+		expCode: 200,
+	}
+
+	t.Cleanup(func() {
+		server.stop()
+		fsCleanup()
+	})
```

Comment:
This is OK, just noting that this should be equivalent:
```suggestion
	t.Cleanup(server.stop)
	t.Cleanup(fsCleanup)
```
....from my understanding at least -- I could be wrong 🤔